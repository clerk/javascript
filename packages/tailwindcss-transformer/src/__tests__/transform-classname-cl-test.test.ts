import { transform } from '../index';

test('empty className', () => {
  const result = transform(
    `const Button: React.FC<ButtonProps> = ({ children, className }) => {
    return <button className='cl-test-123'>{children}</button>
  }`,
    {
      styleCache: new Map(),
    },
  );
  expect(result).toMatchSnapshot();
});
