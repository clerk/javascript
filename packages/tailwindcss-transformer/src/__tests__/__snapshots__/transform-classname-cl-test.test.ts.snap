// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty className 1`] = `
"const Button: React.FC<ButtonProps> = ({ children, className }) => {
    return <button className='cl-test-123'>{children}</button>
  }"
`;
